klog.go

```diff
@@ -43,7 +43,10 @@
 //		Logs are written to standard error instead of to files.
 //	-alsologtostderr=false
 //		Logs are written to standard error as well as to files.
-//	-stderrthreshold=ERROR
+//	-stderrthreshold=INFO
```

Comment:
this might have to land in new release because it's a breaking change.

Reply:
Please check the change inside the code. This is to maintain the pre-existing behavior. 
However, there are no problems to land in a release.